app/views/pregnancies/_new_pregnancy.html.erb

```diff
@@ -11,10 +11,10 @@
   <% end %>
 
   <div class="col-md-2">
-    <%= f.text_field :last_menstrual_period_weeks, label: "LMP" %>
+    <%= f.select :last_menstrual_period_weeks, options_for_select(weeks_options, pregnancy.last_menstrual_period_weeks ), label: 'LMP at intake', autocomplete: 'off' %>
   </div>
   <div class="col-md-2">
-    <%= f.date_field :initial_call_date, label: "Initial Call Date" %>
+    <%= f.select :last_menstrual_period_days, options_for_select(days_options, pregnancy.last_menstrual_period_days ), label: '.', autocomplete: 'off' %>
```

Comment:
This looks great! The weeks and days fields should work well, but can you make sure that you keep in the field for :initial_call_date?